test-e2e/manager-basic.js

```diff
@@ -56,20 +59,90 @@ test('Managing created projects', async (t) => {
   t.ok(project1)
   t.ok(project2)
 
-  t.test('initial settings from project instances', async (st) => {
-    const settings1 = await project1.$getProjectSettings()
-    const settings2 = await project2.$getProjectSettings()
-
-    st.alike(settings1, {
-      name: undefined,
-      defaultPresets: undefined,
-    })
+  await t.test(
+    'initial settings and default config from project instances',
+    async (st) => {
+      const settings1 = await project1.$getProjectSettings()
+      const settings2 = await project2.$getProjectSettings()
+
+      const {
+        presets: defaultPresets,
+        fields: defaultFields,
+        icons: defaultIcons,
+      } = await getExpectedConfig(
+        new URL('../config/defaultConfig.mapeoconfig', import.meta.url).pathname
+      )
+
+      st.is(settings1.name, undefined, 'undefined name for project1')
+      st.is(settings2.name, 'project 2', 'matched name for project2')
+
+      const projectPresets = await project1.preset.getMany()
+
+      st.is(
+        settings1.defaultPresets?.point.length,
+        defaultPresets.length,
+        'the default presets loaded is equal to the number of presets in the default config'
+      )
+
+      st.alike(
+        projectPresets.map((preset) => preset.name),
+        defaultPresets.map((preset) => preset.value.name),
```

Comment:
sort order is not guaranteed, so best to compare sets e.g. `new Set([...projectPresets.map((preset) => preset.name)])`.
Also: I think you have these the wrong way around - `projectPresets` is what has `.value` properties.
Also: test the whole thing e.g. `st.alike(new Set([...projectPresets.map(p => p.value)]), new Set([...expected.presets]))`